### PR TITLE
Console output player command

### PR DESCRIPTION
**Pull request by ChisatoNishikigi73**
## Description
Implement the console output player command

## Issues fixed by this PR

You can better manage the server

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by 2y8e9h22** [comment]
这啥啊，请关闭这个pr。

**Comment by ChisatoNishikigi73** [comment]
> 这啥啊，请关闭这个pr。

我觉得只pr readme的人并没有权利教我做事

File: src/main/java/emu/grasscutter/command/CommandMap.java [hunk 1]
```diff
@@ -22,7 +21,7 @@ public CommandMap(boolean scan) {
         if (scan) this.scan();
     }
 
-    public static CommandMap getInstance() {
+    public static emu.grasscutter.command.CommandMap getInstance() {
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/command/CommandMap.java]: ?

File: src/main/java/emu/grasscutter/command/CommandMap.java [hunk 2]
```diff
@@ -33,7 +32,7 @@ public static CommandMap getInstance() {
      * @param command The command handler.
      * @return Instance chaining.
      */
-    public CommandMap registerCommand(String label, CommandHandler command) {
+    public emu.grasscutter.command.CommandMap registerCommand(String label, CommandHandler command) {
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/command/CommandMap.java]: ?

File: src/main/java/emu/grasscutter/command/CommandMap.java [hunk 3]
```diff
@@ -58,7 +57,7 @@ public CommandMap registerCommand(String label, CommandHandler command) {
      * @param label The command label.
      * @return Instance chaining.
      */
-    public CommandMap unregisterCommand(String label) {
+    public emu.grasscutter.command.CommandMap unregisterCommand(String label) {
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/command/CommandMap.java]: ?

File: src/main/java/emu/grasscutter/command/CommandMap.java [hunk 4]
```diff
@@ -195,6 +194,14 @@ private boolean setPlayerTarget(String playerId, Player player, String targetUid
      * @param rawMessage The messaged used to invoke the command.
      */
     public void invoke(Player player, Player targetPlayer, String rawMessage) {
+        //The console outputs in-game command ,[{Nickname}(Uid:{uid})]
+        String message = "";
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/command/CommandMap.java]: unused variable

**Review by Birdulon — CHANGES_REQUESTED** [review]

**Comment by Birdulon** [comment]
Context for anyone else viewing this: it appears to be another edition of #1019 (related issues #981 #1043)
It logs all command usage, which was previously recommended to just leave as an addon last time the topic was brought up. I've reviewed purely on code style (i.e. please don't change method names to fully-qualified names without reason, please don't introduce unused variables), but I'll leave it up to @Melledy and/or @KingRainbow44 to decide if it should actually be merged after those are fixed.

**Comment by ghost** [comment]
Please make the changes @Birdulon requested and make a config option to control player command logging. Thank you.

**Comment by ChisatoNishikigi73** [comment]
> Please make the changes @Birdulon requested and make a config option to control player command logging. Thank you.

OK,i will

**Comment by ChisatoNishikigi73** [comment]
I changed it

**Review by ghost — APPROVED** [review]

**Review by Birdulon — APPROVED** [review]

**Pull request merged**